frontend/src/action-creator.async.js

```diff
@@ -775,3 +775,12 @@ export const postDownloadFile = (token, guestPassword) =>
       password: guestPassword
     })
   })
+
+export const getGuestUploadInfos = (token) =>
+  fetch(`${FETCH_CONFIG.apiUrl}/public/guest-upload/${token}`, {
```

Comment:
frontend async calls must use the `fetchWrapper()` function instead of `fetch()`